run-tests

```diff
@@ -58,5 +60,7 @@ def run_once():
 if __name__ == '__main__':
     import sys
 
+    install_govuk_assets('app')
```

Comment:
This nearly doubles the time it takes to run tests on Travis CI. Can the govuk_template be mocked or something instead?